packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -192,6 +192,46 @@ export class SummarizerNode implements IRootSummarizerNode {
 		return { summary: result.summary, stats: result.stats };
 	}
 
+	public validateSummary() {
+		this.validateSummaryCore(false);
+	}
+
+	protected validateSummaryCore(parentSkipRecursion: boolean) {
```

Comment:
Can you add TSDoc comments for this, especially what `parentSkipRecursion` means?  Same for `completeSummaryCore` below

Reply:
yep, will do